console/src/app/modules/policies/login-policy/login-policy.component.html

```diff
@@ -189,7 +189,7 @@ <h3>{{ 'DESCRIPTIONS.SETTINGS.LOGIN.LIFETIMES.TITLE' | translate }}</h3>
     [title]="'DESCRIPTIONS.SETTINGS.LOGIN.LIFETIMES.MULTI_FACTOR_CHECK.TITLE' | translate"
     [description]="'DESCRIPTIONS.SETTINGS.LOGIN.LIFETIMES.MULTI_FACTOR_CHECK.DESCRIPTION' | translate"
   >
-    <cnsl-form-field class="lifetime-form-field" label="Multi Factor Check Lifetime" required="true">
+    <cnsl-form-field class="lifetime-form-field" label="Multifactor Check Lifetime" required="true">
```

Comment:
I think there are different way on how to write multifactor, I had a quick look on what we have in our repository. and I can see that we are a little inconsistent, but from my impresseion we never use Multifactor as one worde.
Its either Multi-factor, or multifactor.

```suggestion
    <cnsl-form-field class="lifetime-form-field" label="Multi-factor Check Lifetime" required="true">
```